src/backend/distributed/shared_library_init.c

```diff
@@ -746,6 +746,16 @@ RegisterCitusConfigVariables(void)
 		0,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.allow_executor_swap",
```

Comment:
This name is not very descriptive.

I'd have a preference for `task_tracker_query_log_level`, analogous to `multi_task_query_log_level`, or perhaps `repartition_query_log_level`.

An alternative would be `enable_task_tracker`, but we'd like to preserve the behaviour of `SET citus.task_executor_type TO 'task-tracker'` and it would be a little confusing if task-tracker is not enabled but you can still run queries with it.

Reply:
Or maybe `enable_repartition_joins` with values 'on', 'off' (default).

Reply:
changed to enable_repartition_joins